components/director/internal/domain/runtime/repository_test.go

```diff
@@ -251,3 +326,7 @@ func mockDatabase(t *testing.T) (*sqlx.DB, sqlmock.Sqlmock) {
 
 	return sqlxDB, sqlMock
 }
+
+func convertIntToString(number int) string {
```

Comment:
don't we have a method for that in pagination package?

Reply:
We have something similar `EncodeOffsetCursor`, but using this, may be misleading, because this method return cursor to to the next page, so I am going to rename it to `EncodeNextOffsetCursor`